### ISSUE TITLE
Chapter 11, model.save() returned NotImplementedError

### ISSUE THREAD

**Issue by altayg** [deepchem/DeepLearningLifeSciences#16]
This notebook worked well except the last line:

https://github.com/deepchem/DeepLearningLifeSciences/blob/master/Chapter11/chapter_11_02_erk2_graph_conv.ipynb

model.save() returned as below. Any suggestion on this?

---------------------------------------------------------------------------
NotImplementedError                       Traceback (most recent call last)
<ipython-input-23-fd5dede08c85> in <module>()
----> 1 model.save()

~/.conda/envs/deepchem/lib/python3.5/site-packages/deepchem/models/models.py in save(self)
    107     Each subclass is responsible for overriding this method.
    108     """
--> 109     raise NotImplementedError
    110 
    111   def fit(self, dataset, nb_epoch=10, batch_size=50, **kwargs):

NotImplementedError:

**Comment by peastman** [comment]
I take it you're using the latest DeepChem code from github?  The current code in this repository is written to work with DeepChem 2.2.  As soon as 2.3 is released (hopefully very very soon), I'll post updated code here that works with it.

**Comment by altayg** [comment]
It seems I have 2.1.1
Is the default installation 2.2 ?

**Comment by peastman** [comment]
Yes, 2.2 should be the default.  But if in doubt, follow the installation instructions at https://github.com/deepchem/deepchem.

**Comment by altayg** [comment]
When I use 
pip3 install joblib pandas sklearn tensorflow pillow deepchem

it installs '2.1.1'

I install it in an environment with Python 3.7.4

**Comment by PatWalters** [comment]
This notebook works perfectly for me.  I'm using a conda environment that I
created with

conda create -n deepchem
conda activate deepchem
conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem=2.2.0

My environment is using Python 3..6.7

Pat

**Comment by altayg** [comment]
Thanks.
Initially, I had installed with this from git.
https://github.com/jordenkass/deepchem-for-dummies
That version gave the error.

Upon your suggestion, I tried all the suggested methods an keep seeing the version as 2.1.1 after many installations. 

I then found this issue: https://github.com/deepchem/deepchem/issues/1626
It looks 2.1.1 is actually 2.2. 

I then re-run the notebook and it runs perfectly now.
